ui/oopsyraidsy/data/04-sb/raid/o4n.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+// O4N - Deltascape 4.0 Normal
+[{
+  zoneRegex: {
+    en: /^Deltascape \(V4\.0\)$/,
+  },
+  damageWarn: {
+    'Blizzard III': '24BC', // Targeted circle AoEs, Exdeath
+    'Empowered Thunder III': '24C1', // Untelegraphed large circle AoE, Exdeath
+    'Zombie Breath': '24CB', // Conal, tree head after Decisive Battle
+    'Clearout': '24CC', // Overlapping cone AoEs, Deathly Vine (tentacles alongside tree head)
+    'Black Spark': '24C9', // Exploding Black Hole
+  },
+  triggers: [
+    {
+      id: 'O4N Doom', // Kills target if not cleansed
+      gainsEffectRegex: gLang.kEffect.Doom,
+      deathReason: function(e) {
+        return { type: 'fail', name: e.targetName, reason: { en: 'Cleansers missed Doom!' } };
+      },
+    },
+    {
+      id: 'O4N Vacuum Wave', // Short knockback from Exdeath
+      damageRegex: '24B8',
+      deathReason: function(e) {
+        return { type: 'fail', name: e.targetName, reason: { en: 'Pushed off!' } };
+      },
+    },
+    {
+      // Empowered Fire III inflicts the Pyretic debuff, which deals damage if the player
+      // moves or acts before the debuff falls. Unfortunately it doesn't look like there's
+      // currently a log line for this, so the only way to check for this is to collect
+      // the debuffs and then warn if a player takes an action during that time. Not worth it
+      // for Normal.
+      id: 'O4N Standard Fire',
```

Comment:
Does this do enough damage such that this is something people won't do? I know in savage you definitely had to stop, but I can't remember if this is a "heal through it" pyretic in normal mode.

Reply:
Not for tanks, and for Shadowbringers-geared players, it's probably irrelevant. (Queues are too long currently for me to test it as damage.) I do know that even in the later tiers of Stormblood it was dangerous to ignore the debuff if we didn't know the healers would be on top of things.

Reply:
Yeah, I think triggers should generally assume synced ilvl.  If folks are unsynced and rolling through it, they can ignore triggers as they see fit.